src/sentry/models/release.py

```diff
@@ -138,12 +138,16 @@ def _convert_build_code_to_build_number(build_code):
         if build_code is not None:
             try:
                 build_code_as_int = int(build_code)
-                if build_code_as_int >= 0 and build_code_as_int.bit_length() <= 63:
+                if ReleaseModelManager.validate_bigint(build_code_as_int):
                     build_number = build_code_as_int
             except ValueError:
                 pass
         return build_number
 
+    @staticmethod
+    def validate_bigint(value):
+        return isinstance(value, int) and value >= 0 and value.bit_length() <= 63
```

Comment:
This doesn't exist as a util somewhere else?

Reply:
I don't think it does.